### ISSUE TITLE
Mismatched Coordinate Scales of Meshes in MMWHS MR Data

### ISSUE THREAD

**Issue by mihoo787** [ArjunNarayanan/LinFlo-Net#9]
I used the MR data from the MMWHS dataset. First, I performed data augmentation on the data to obtain 160 training samples and 40 validation samples. After training the model, I used the "predict_udf_test_meshes.py" script to get the predicted meshes for the validation set. Then, I passed the predicted meshes and the ground truth meshes through the "compute_test_mesh_distance.py" script to obtain the Hausdorff distances. However, when I got the Hausdorff distances, I found that the coordinate values of the ground truth meshes are very small, ranging between 0 and 1, while the coordinate values of the predicted meshes are quite large, approximately between -200 and 200. This has prevented me from achieving the experimental results presented in the paper. Could you please tell me how I can solve this problem? I would be extremely grateful if you could help me with this issue.
